### DISCUSSION TITLE
HELP-Merge multiple XML into one training set

### DISCUSSION THREAD

**Discussion by llsnnn** [deepmodeling/dpdata#518] [Q&A]
I wanted to use dpdata to merge multiple datasets (Same structure), but I didn't understand. For example, I have 10 XML files with a total of 100000 frames of data here, and now I want to generate a dataset and take 2000 of them as the test set. Sorry for not resolving this issue from the quick tutorial.

**Comment by llsnnn** [comment]
could i use this method？ #438

**Reply by llsnnn** [reply]
This method is ok! Although I don't know why one group couldn't be merged ....
